libs/components/modals/testing/src/modal/modal-harness.ts

```diff
@@ -83,4 +83,15 @@ export class SkyModalHarness extends SkyComponentHarness {
     const modal = this.#getModal();
     return (await modal).hasClass('sky-modal-full-page');
   }
+
+  /**
+   * Whether the modal has {@link SkyModalIsDirtyDirective.isDirty} set to dirty.
+   */
+  public async isDirty(): Promise<boolean> {
+    const modalHost = await this.host();
+    const isDirtyAttribute = await modalHost.getAttribute(
```

Comment:
I meant to ask about this in hopes that maybe you all would have some ideas. I agree that this is definitely not ideal since it relies on an Angular implementation detail, but I wasn't sure how else to test the value of the directive's input. My hope was that if Angular changed thing, we'd only have to update this one function for everyone.

It sounds like maybe you'd prefer if the directive added a new element to the directive that contained the @Input's value. I think I can do that and then test for it here.